cmd/wit-bindgen-go/cmd/generate/generate.go

```diff
@@ -159,5 +182,32 @@ func action(ctx context.Context, cmd *cli.Command) error {
 		}
 	}
 
+	witDir := filepath.Join(out, "wit")
```

Comment:
This seems like a lot of complexity. What is being written to the `wit` directory? WIT or wasm?

Reply:
the WIT package is output to `<out>/wit/webassembly.wit.wasm`

Reply:
In TinyGo you can just do

`tinygo build -target=wasip2 -wit-package wit/webassembly.wit.wasm -wit-world proxy main.go`